test/scenarios/create_dir_with_utf8_character/local/darwin_hfs+.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "type": "addDir",
+    "path": "Partages reçus",
```

Comment:
FYI, this path is encoded using NFD while the path for APFS is encoded using NFC.